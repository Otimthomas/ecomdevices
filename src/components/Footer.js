import React from "react";
import styles from "styled-components";

function Footer() {
  return (
    <Wrapper>
      <h3>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Tempore nemo
        quod numquam,
      </h3>
      <p>Lorem ipsum dolor sit amet consectetur adipisicing</p>
      <div className="copyright__text">
        <p>Â©Designs {new Date().getFullYear()}</p>
      </div>
    </Wrapper>
  );
}

export default Footer;

const Wrapper = styles.footer`
  background-color: rgb(51,51,51);
  color: #ffffff;
  text-align: center;
  padding: 20px;

  .copyright__text{
    margin-top: 25px;
  }
`;
