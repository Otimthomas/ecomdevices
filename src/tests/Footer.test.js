import Footer from "../components/Footer";
import renderer from "react-test-renderer";

describe("<Footer />", () => {
  it("matches the snapshot", () => {
    let tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
