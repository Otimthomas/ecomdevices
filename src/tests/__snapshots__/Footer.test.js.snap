// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches the snapshot 1`] = `
<footer
  className="sc-bdvvtL coUWpe"
>
  <h3>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Tempore nemo quod numquam,
  </h3>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing
  </p>
  <div
    className="copyright__text"
  >
    <p>
      Â©Designs 
      2022
    </p>
  </div>
</footer>
`;
